script.js

```diff
@@ -0,0 +1,12 @@
+const hamburger = document.querySelector('.hamburger');
+const navMenu = document.querySelector('.nav__menu');
+
+hamburger.addEventListener('click', () => {
+  hamburger.classList.toggle('active');
+  navMenu.classList.toggle('active');
+});
+
+document.querySelectorAll('.nav__link').forEach((n) => n.addEventListener('click', () => {
+  hamburger.classList.remove('active');
+  navMenu.classList.remove('active');
```

Comment:
- [ ] Great job working on the menu but I think when I press on about it doesn't take me to the about section and so on for all the links but in the requirements rach link it should take me to that section so Portfolio it sould take me to the projects and about to about section and contact to contact section so kindly fix that 💯💪🏻 and remember that the mobile menu disappears after taking the user to the specific section 😊📌💯🔥🔥🔥 

![image](https://user-images.githubusercontent.com/58553711/215626244-7a2e40d4-dbb8-4bd5-9f09-d4cffb550fb2.png)